jupyter-book/trajectories/rna_velocity.ipynb

```diff
@@ -15,53 +15,117 @@
    ]
```

Comment:
<p>Do you call that the "EM model" or the "Dynamic model". I'm not deep into the topic, but I had the impression that people mostly refer to it as the "dynamic model"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/multimodal-single-cell-best-practices/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Yes, people mostly call it dynamical model. I personally prefer <em>EM model</em> because there are a few models now that are based on the dynamical system. Happy to change it if you prefer <em>dynamical model</em>.</p>

Reply:
All right, let's roll with _EM model_ then. You can also consider that some people refer to it as the _dynamical model_? Totally optional.